public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js

```diff
@@ -249,6 +259,15 @@ function ContextMenu (uiContextMenu) {
         });
     }
 
+    function removeLabelAndUpdateUI(labelName, labelTags) {
+        $tags.each((index, tag) => {if (tag.innerText == labelName) {tag.style.backgroundColor = "white"; } });
+        self.labelTags.forEach(tag => {
+            if (tag.tag == labelName && labelTags.includes(tag.tag_id)) {
+                labelTags.splice(labelTags.indexOf(tag.tag_id), 1);
```

Comment:
Similar to how it is done above, we want to log when a tag is deselected. So could you log 'ContextMenu_TagAutoRemoved' here? (like above, but "AutoRemoved" instead of "Removed")

Reply:
You logged "TagRemoved" instead of "TagAutoRemoved"

Reply:
Maybe you could take in an `autoRemoved` boolean parameter so you know whether to log autoremoved to just removed.

Reply:
Done (log "TagAutoRemoved" now)
"TagAutoRemoved" is used only once. Maybe we can do what you suggested for "TagRemoved"/"TagAdded"? Like switch between on and off to decide if the tag is removed or added?